### ISSUE TITLE
Restart after "Update" hangs in splash screen if executable location is write protected/needs admin privileges

### ISSUE THREAD

**Issue by kunom** [MuhammedKalkan/OpenLens#49]
I just applied this incoming version update by hitting the green update button in the top left corner of the main window.

On restart, the splash screen "loading..." did not close, the main window opened in the background. This was reproducible after killing and try again. It resolved only after I tried again, this time with launching OpenLens with "run as administrator".

**Comment by MuhammedKalkan** [comment]
Try the new release update and see if this exists

**Comment by kunom** [comment]
I just performed the update from `6.1.2` to `6.1.10`. After pressing the green "Update" button, the OpenLens main window disappeard and did not any more reappear again. I then opened it manually, no more issues with "loading...", but then the program still was a bit dysfunctional (meaning: all my existing cluster registrations were gone). There was around 10% CPU load and some _Trusted Installer_ was running in the background. Quitting OpenLens and reopening for a few times did not help. Then I was interrupted, and now (2h later), OpenLens behaves correctly after startup.

So, in the end, I am not fully sure whether everything is resolved, but I definitively did not anymore need to run OpenLens as administrator to get the update process to complete. From my side, this bug can be closed.

**Comment by JanSchep** [comment]
I have the same issue with an endless loading splash screen. A fresh install and starting as administrator did not help. In rar cases I was able to see my configured clusters after some restarts. But additionally kubexec did not work anymore with the actual version. I use windows and wsl2. I now downgraded to version 6.0.2 and voila, all is working again like charm. App is loading after 10sec and I can use kubeexec again. I have no clue if these issues are related to the source code or on the binary building workflow. I think most likely is that there were some breaking source code changes.
